### PR TITLE
Enchanting.Enchanting version 0.2.4.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103044)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230420231530

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
Resetting approval since it is so old; Re-reviewing now
[Policy] Reset Feedback

File: manifests/e/Enchanting/Enchanting/0.2.4.3/Enchanting.Enchanting.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Enchanting.Enchanting
+PackageVersion: 0.2.4.3
+PackageLocale: en-US
+Publisher: Enchanting
+PackageName: Enchanting
+License: Propeitaru
```
Comment (Trenly) [inline manifests/e/Enchanting/Enchanting/0.2.4.3/Enchanting.Enchanting.locale.en-US.yaml:9]:
```suggestion
License: Propeitary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/e/Enchanting/Enchanting/0.2.4.3/Enchanting.Enchanting.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/e/Enchanting/Enchanting/0.2.4.3/Enchanting.Enchanting.locale.en-US.yaml:9]: Also, still need to add the missing dependency

**Pull request closed without merging**